### PR TITLE
Re-add FAQ with expanded answers

### LINKED ISSUE #1951: Expand FAQ
Expand answers to FAQ based on skeleton added in #1914, and add FAQ page back (was removed in https://github.com/fluxninja/aperture/pull/1945).

cc @harjotgill @sudhanshu456

### PR DESCRIPTION
**Pull request by krdln**
Readded the FAQ skeleton (https://github.com/fluxninja/aperture/issues/1951)
and expanded some answers.

~~Some things I'm unsure about, added comments/TODOs.~~

[:framed_picture: Preview](https://github.com/fluxninja/aperture/blob/faq-reborn/docs/content/faq.md)

Resolves #1951 

### Summary by OpenAI

**Documentation:**
- Expanded FAQ section in `docs/content/faq.md` with new questions and answers related to Aperture's usage, rate limiting, request rejection, flow label uniqueness, delays in server availability, and Kubernetes cluster requirements.

**Style:**
- Updated `.github/styles/Vocab/FluxNinja/accept.txt` with additional vocabulary.

> 🎉 A wealth of knowledge we now share,
> In our FAQ, answers everywhere! 📚
> Rate limits, requests, and more,
> FluxNinja's wisdom, we adore! 💡🥳

### CODE REVIEW

File: docs/content/faq.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  accept/reject request immediately based on workload priorities and other
+  factors.
+
+### If Aperture is rejecting or queuing requests, how will it impact the user experience?
+
+Queuing requests should not affect user experience (apart of increased latency).
+When it comes to rejecting requests, clients (whether it's frontend code or some
+other service) should be prepared to receive 429 Too Many Requests or 503
+Service Unavailable response and react accordingly. We're working on a library
+to make it easy to handle these scenarios and provide nice UX for end users.
```
Comment (krdln) [inline docs/content/faq.md:56]: Should we mention this library? How far away is it?

File: docs/content/faq.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Remember that while receiving 503 by some of users may seem like a thing to
+avoid, if such case occurs overload is already happening Aperture is protecting
+your service from unhealthy state (eg. crashing) and thus affecting even more
+users.
+
+### How can we ensure the uniqueness of requests for the flow label when using Aperture?
+
+Aperture uses the flow label to identify the requests.
+
+TODO (I don't get the question tbh)
```
Comment (krdln) [inline docs/content/faq.md:67]: @sudhanshu456 I don't understand this question

Comment (sudhanshu456) [reply docs/content/faq.md:67]: Resolved over Slack.

File: docs/content/faq.md (hunk 1 above)
Comment (krdln) [reply docs/content/faq.md]: After talking with @hdkshingala , decided to remove this mention. We can readd it later https://github.com/fluxninja/graceful-js/issues/5

File: docs/content/faq.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+colocating agents with services, it's a single RPC call within a single node.
+
+### If we already have circuit breakers and rate limiting in EnvoyProxy, what are the benefits of using Aperture? {#envoy-rate-limit}
+
+While Envoy does have some local and non-local rate limiting capabilities, there
+are still benefits of using Aperture:
+
+- Aperture Rate limiter allows dynamically configuring Rate Limiter parameters
+  via signals from Policy.
+- Ability to configure global rate limiting without configuring any external
```
[the remark this answers is not in the document]
Comment (krdln) [reply docs/content/faq.md:33]: (decided to not apply)

File: docs/content/faq.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Aperture uses the flow label to identify the requests.
+
+TODO (I don't get the question tbh)
+
+### How does Aperture address the issue of delays in servers becoming available and reaching a healthy state, particularly in the context of auto-scaling?
+
+As Aperture observes the system, it can detect early sign of overload and can
+take necessary actions to prevent the system from becoming unhealthy. Therefore,
+server gets enough time to reach a healthy state.
```
[the remark this answers is not in the document]
Comment (krdln) [reply docs/content/faq.md:76]: not applying

File: docs/content/faq.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+- With SDKs, it's possible to explicitly pass Flow Labels to the Check call.
+- Proxy-based integrations can use a [Classifier][] to define new Flow Labels.
+
+See the [Flow Label][] page for more details.
+
+### How does Aperture address the issue of delays in servers becoming available and reaching a healthy state, particularly in the context of auto-scaling?
+
+As Aperture observes the system, it can detect early sign of overload and can
+take necessary actions to prevent the system from becoming unhealthy. Therefore,
+the server gets enough time to reach a healthy state.
```
[the remark this answers is not in the document]
Comment (krdln) [reply docs/content/faq.md:76]: not applying

**Comment by krdln** [comment]
I had interesting discussion with chatgpt about colocating vs collocating:

> The correct spelling in this context is indeed "colocating." "Colocating" is the present participle form of the verb "colocate," which means to place or locate together. Therefore, "colocating agents with services" is the appropriate phrase in your sentence.
> 
> "Collocating" is a term related to linguistics and has a different meaning from the context we were discussing.
> 
> In linguistics, "collocating" refers to words or terms that tend to occur together or are often found in close proximity to each other in a given language or text. It pertains to the concept of collocation, which explores the patterns and relationships between words.
>

**Pull request merged**